index.html

```diff
@@ -0,0 +1,85 @@
+<!Doctype HTML>
+<html lang="en">
+<head>
+    <title>CYF-WDT</title>
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Philosopher" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="body-content">
+            <h1 class="logo" id="headLogo">CYF</h1>
+            <nav>
+                <ul class="links">
+                    <li><a href="#">Menu 1</a></li>
+                    <li><a href="#">Menu 2</a></li>
+                    <li><a href="#">Menue 2</a></li>
+                </ul>
+            </nav>
+            <article>
+                <h1 class="about-cyf">About CYF</h1>
```

Comment:
good class names - you are very consistent (using lower case and dashes all the time). This is very good and important.